qcengine/programs/gcp.py

```diff
@@ -168,10 +167,12 @@ def build_input(
         # Need 'real' field later and that's only guaranteed for molrec
         molrec = qcel.molparse.from_schema(input_model.molecule.dict())
 
+        executable = self._defaults["name"].lower()
+
         if method == "FILE":
-            command = ["gcp", "gcp_geometry.xyz", "-local"]
+            command = [executable, "gcp_geometry.xyz", "-local"]
```

Comment:
At least `mctc-gcp` does not support a `-local` flag and I don't find this flag in the CLI of the old driver as well.
The option to get the parameter file would be `--level file` for both new and old driver.

Reply:
I found `-local` in help for classic. My guess is that `method == "FILE"` is prep for `.gcppar` file and not presently used. Does this sound right, @hokru? And if the r2SCAN-d* methods need mctc, is it more likely that using a JSON interface is the next step, rather than `.gcppar`?

```
(gcp) psilocaluser@bash:psinet:/psi/gits/QCEngine: (mctcgcp) gcp -h
  ___________________________________________ 
                                              
 |              **  g C P  **                |
 |  a geometrical counterpoise correction    |
 |     H.Kruse J.G.Brandenburg S.Grimme      |
 |          Version 2.02 Nov 2016            |
 |                                           |
  ___________________________________________ 
 
 Please cite work done with this code as:
 H. Kruse, S. Grimme J. Chem. Phys. 136, 154101 (2012)
 DOI: 10.1063/1.3700154 
 For the periodic version, please also cite:
 J. G. Brandenburg, M. Alessio, B. Civalleri, M. F. Peintinger
 T. Bredow, S.Grimme J. Phys. Chem. A 117, 9282-9292 (2013).
 DOI: 10.1021/jp406658y 
 
 
                                   
  *    ----   HELP    ----    *    
                                    
  gcp <coordinates> -level <method>   
                                    
  example:                                 
  -> gcp water.xyz -level dft/631gd     
  -> gcp POSCAR -pbc -level hf/svp 
                                   
                                   
  Valid input coordinates are      
    - TURBOMOLE files (in bohr)    
    - XMOL files (in angstrom)     
    - VASP POSCAR files (in angstrom)
    - CRYSTAL STRUC.INCOOR files (in bohr)
                                   
  available parametrisation for <method>        
       HF/MINIS       DFT/MINIS
       HF/MINIX       DFT/MINIX
       HF/SV          DFT/SV 
       HF/def2-SV(P)  DFT/def2-SV(P)
       HF/def2-SVP    DFT/def2-SVP
       HF/DZP         DFT/DZP
       HF/def-TZVP    DFT/def-TZVP
       HF/def2-TZVP   DFT/def2-TZVP
       HF/631Gd       DFT/631Gd
       HF/def2-TZVP   DFT/def2-TZVP
       HF/cc-pVDZ     DFT/cc-pVDZ
       HF/aug-cc-pVDZ DFT/aug-cc-pVDZ
         -            DFT/SV(P/h,c)
         -            DFT/LANL
                                   
  note: <method> does not need to be          
        capitelized                           
                                   
  command line options:            
     -h                       this print out   
     -pbc                     use periodic version   
     -level <string>    specify <method>   
       if <method> = file or left empty a parameter   
       file from ~/.gcppar.$HOSTNAME will be read in    
     -l                       same as -level   
     -grad                    request gradient 
     -stress                  request cell gradient 
     -noprint                 surpress output   
     -parfile                 print <gcp.param> 
     -local                   use  .gcppar in work-dir 
     -hess                    request hessian  
     -test                    stop after parameter setup  
     -v                       verbose output: print gradient to stdout  
                              instead of gcp_gradient     
     -vasp                    define input as VASP POSCAR file 
     -crystal                 define input as CRYSTAL STRUC.INCOOR file 
                                   
                                   
  *    ----   ****    ----    *    
 
     normal termination of help
```

Reply:
looks like FILE is in use.

Reply:
the `-local` option in the classic version did nothing. Not sure what happend there. `-level file` is the same as specifying no `-level`, ie reading from `~/.gcppar.$HOSTNAME`